tests/lean/run/builtinInitStx.lean

```diff
@@ -0,0 +1,3 @@
+@[inline]
```

Comment:
Please add documentation here that describes this test! The preferred format is described [here](https://github.com/leanprover/lean4/blob/master/doc/dev/testing.md#writing-good-tests).